### PR TITLE
refactoring, some bluetooth (untested)

### PR DESCRIPTION
**Pull request by b4yuan**
Bluetooth is untested but App.js builds

### CODE REVIEW

**Review by jtrowe08 — DISMISSED** [review]
Looks great.

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { StyleSheet, Text, View } from 'react-native';
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+      <Text>TEST TEST TESTOpen up App.js to start working on your app!</Text>
```
Comment (GraysonNocera) [inline src/App.js]: Remove?

File: src/services/bluetooth/bluetoothReceiver.js
Comment (GraysonNocera) [inline src/services/bluetooth/bluetoothReceiver.js:1]: Looks good dude. How we will receive data after initializing? My hope is that we can find some package or something that essentially just calls a callback function everytime data is received and we can handle it accordingly based on what type of data it is.

**Comment by GraysonNocera** [comment]
Should we try to do TDD to some extent? As in writing testable code the first time and actual tests for it before merging. I know that went out the window in 461 but if we wanna put this on our resume and make it all nice and stuff that would be epic

**Review by jtrowe08 — APPROVED** [review]
epic

**Review by khauersp — APPROVED** [review]

**Pull request merged**